runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/base/README.tpl.qute.md

```diff
@@ -0,0 +1,33 @@
+{#include readme-header /}
+
+## Hello-World application
+
+Hilla provides the `init-app` maven goal to create a Hello-World endpoint and view, and also required front-end
+dependencies and TypeScript configurations to boost development.
+However, this currently doesn't work out-of-the-box with the Quarkus-Hilla extension.
+
+To be able to run the `init-app` goal you need some temporary workarounds in your project:
+
+- Add a simple Java class in your base package folder with a comment containing the `@SpringBootApplication` text.
+  ```java
+    package com.acme;
```

Comment:
From where comes the com.acme?

Reply:
`com.acme` is wrong, it should be `org.acme`.

The documentation states that if you provide source code, it should be in the  'org.acme` package, so I used it in the readme, but it can probably be replaced with the `project.package-name` variable. I'll fix it

https://quarkus.io/guides/extension-codestart#org-acme-package

Reply:
done